db/migrate/20230907050143_add_flag_for_cell_facet_filtering.rb

```diff
@@ -0,0 +1,11 @@
+class AddFlagForUserDeUpload < Mongoid::Migration
```

Comment:
Blocker: I can't test the functionality in this PR because this migration won't run.  The name of the file and the class of the migration have to be the same.  E.g. `20230907050143_add_flag_for_cell_facet_filtering.rb` would need to have a class name of `AddFlagForCellFacetFiltering`.  Since the class name is actually `AddFlagForUserDeUpload` it barfs with:
```
NameError: uninitialized constant AddFlagForCellFacetFiltering
```

Reply:
oh geeze sorry - fixed [70de49a](https://github.com/broadinstitute/single_cell_portal_core/pull/1878/commits/70de49a11166461740cf9e9fab050ed6bb81d690)